kuma/wiki/tests/test_views.py

```diff
@@ -2623,8 +2616,7 @@ def test_midair_section_collision_ajax(self):
             'wiki.document_revisions',
             kwargs={'document_path': rev.document.slug})
         midair_collission_error = (unicode(MIDAIR_COLLISION) % {'url': history_url}).encode('utf-8')
-        ok_(midair_collission_error in json.loads(resp.content)['error_message'],
-            "Midair collision message should appear")
+        assert midair_collission_error in json.loads(resp.content)['error_message'], "Midair collision message should appear"
```

Comment:
Again, since the line exceeds the PEP8 line-length limit and the assert error message doesn't really add any value, I would suggest dropping the error message:
```python
assert midair_collission_error in json.loads(resp.content)['error_message']
```